bin/vhost

```diff
@@ -174,10 +175,33 @@ fi
 #═════════════════════════════════════════════════════════════════════════════════════════════════
 if [ "$command" = "install" ]; then
   require_config_dir
+
   ip=127.0.0.1
-  hosts=$( jq -rM '.hosts | join(" ")' $config_file )
+  # Merge local config if any
+  if test -f "$local_config_file"; then
+    ptask "Using local config"
+    config=$(jq -s '.[0] * .[1]' $config_file $local_config_file)
```

Comment:
Cool, so that's how you merge with jq! I had to look up what `*` operator means. Found it [here](https://hyp.is/YvtpPlVgEe2sFP97nEGWnQ/stedolan.github.io/jq/manual/v1.6/)

Reply:
It looks like there's a limitation of `*`: it doesn't merge arrays. :frowning: 

So if you had a `hosts` array defined in both, the `hosts` from local would completely override the base/shared version.

I found a workaround, but it requires hard-coding/repeating for each key that has an array value:

```
jq -s '.[0] * .[1] * {hosts: (.[0].hosts + .[1].hosts)}' ...
```

Do you know of a better way by change?

Reply:
We actually *need* the array in the case of the `hosts` to be overridden. So I'm not sure what to suggest really.

Reply:
So, for the way it was being used before, it was a feature 😄

Reply:
Okay, good to know. I guess I just hadn't thought of a use case where you'd want to override `hosts`, but I can see how that could be useful hypothetically. And it's easy enough to just repeat the same list in your local file if you don't want any entries removed.

Great, I guess we don't need to worry about merging arrays then! — at least not unless we later add one that we do want to be merged instead of replaced, in which case maybe we just explicitly list them like in my example.

(BTW, I did come across a possible solution to the general problem when you _do_ want to merge objects / concatenate arrays: https://stackoverflow.com/questions/66907565/how-to-deep-merge-2-files-with-jq-for-arrays)

Reply:
Nice. Yeah, for our app, we each have to issue certs for domains we have configured, so we need to be able to setup the host mappings. I can technically give you a url that works from anywhere in the world that serves the app from my local dev machine

Reply:
That sounds pretty cool! Some day you should share more about how you managed to set that up..